tests/target/issue_6069.rs

```diff
@@ -0,0 +1 @@
+trait Foó: Bar {}
```

Comment:
Can you add a comment here to better explain the issue. Maybe something like what you added to the description of #6069:

```suggestion
// The `ó` in `Foó` below is two codepoints, ASCII `o` followed by `U+0301` (COMBINING ACUTE ACCENT).
// It NFC-normalizes to `ó` (`U+00F3` LATIN SMALL LETTER O WITH ACUTE).
// The chars for the trait bound actually look like: `trait Foo\u{301}: Bar {}`
trait Foó: Bar {}
```